cmd/launcher/desktop.go

```diff
@@ -98,7 +99,14 @@ func runDesktop(args []string) error {
 		return err
 	}
 
-	menu := menu.New(logger, *flhostname, *flmenupath)
+	td := menu.NewTemplateData(
+		menu.WithVersion(version.Version()),
+		menu.WithOSQueryVersion("TODO"),
```

Comment:
I've put TODOs here since I haven't yet explored how difficult it is to get some of these things within launcher desktop process. Can always omit them for this PR and add them in as-needed in the future.